src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -875,3 +879,20 @@ registerEditorCommand(new SuggestCommand({
 		primary: KeyMod.Shift | KeyCode.Tab
 	}
 }));
+
+
+registerEditorAction(class extends EditorAction {
+
+	constructor() {
+		super({
+			id: 'editor.action.resetSuggestSize',
+			label: nls.localize('suggest.reset.label', "Reset Suggest Widget Size"),
```

Comment:
We do not use the word "Widget" in all our commands.
We only have one command with suggest and it is called
"Trigger Suggest"

Due to that I suggest we call this
"Reset Suggest Size"

Reply:
Settings is big in talking about "suggest widget" that why I am using it here